src/notifications/tests/test_models.py

```diff
@@ -63,3 +65,120 @@ def new_notification(sender, **kwargs):
 
         notes_unread = Notification.objects.all_unread(self.student)
         self.assertEqual(notes_unread.count(), 1)
+
+
+class NotificationModelStrTest(TenantTestCase):
+    """  
+        This test class is specialized on testing the __str__ method of NotificationModel
+    """ 
+    # HELPER FUNCTIONS
+
+    def create_notification(self, text="Sample Text") -> Notification:
+        """ 
+            Quick helper function that spits out different notifications depending on text var
+        """
+        comment = baker.make("comments.Comment", text=text)
+        notification = Notification.objects.create(
+            recipient=self.student,
+            sender_object=self.admin,
+            verb="",
+
+            action_object=comment,
+            action_object_id=comment.pk,
+            action_content_type=self.comment_content_type,
+
+            target_object=self.quest_submission,
+            target_object_id=self.quest_submission.pk,
+            target_content_type=self.quest_submission_content_type,
+
+            font_icon="<i class='fa fa-info-circle'></i>",
+        )
+        return notification
+
+    def notification_start_text(self, notification_id) -> str:
+        """  
+            Recreates string tag for testing purposes. Equivalent to: str(notification) up until action_object
+            '{notification_start_text(id)}{action_object}{notification_end_text}' <=> str(notification)
+        """
+        site_url = 'https://tenant.test.com'
+        notification_url = reverse('notifications:read', args=[notification_id])
+        submission_url = reverse('quest_manager:submission', args=[self.quest_submission.pk])
+        chars = f"{self.admin.username}  <a href='{site_url}{notification_url}?next={submission_url}'> <em>{self.quest.name}</em> with \""
```

Comment:
Seems weird to manually regenerate the string... why not just call str(notification) since that's what you're actually testing?  I don't understand the flow of these tests.